File: src/main/java/dev/discordmk/neutron/components/NeutronMenuBar.java
package dev.discordmk.neutron.components;

import javax.swing.*;

public class NeutronMenuBar extends JMenuBar {
    public NeutronMenuBar() {
        JMenu fileMenu = new JMenu("File");
        
        
        JMenuItem openItem = new JMenuItem("Open");
        JMenuItem saveItem = new JMenuItem("Save");
        JMenuItem exitItem = new JMenuItem("Exit");

        fileMenu.add(openItem);
        fileMenu.add(saveItem);
        fileMenu.addSeparator();
        fileMenu.add(exitItem);
        
        add(fileMenu);

        add(Box.createHorizontalGlue());

        JButton saveButton = new JButton("Save");
        
        add(saveButton);
    }
}

File: src/main/kotlin/dev/tonimatas/discordmk/components/NeutronMenuBar.kt
package dev.tonimatas.discordmk.components

import javax.swing.*

class NeutronMenuBar : JMenuBar() {
    init {
        val fileMenu = JMenu("File")

        val openItem = JMenuItem("Open")
        val saveItem = JMenuItem("Save")
        val exitItem = JMenuItem("Exit")

        fileMenu.add(openItem)
        fileMenu.add(saveItem)
        fileMenu.addSeparator()
        fileMenu.add(exitItem)

        add(fileMenu)

        add(Box.createHorizontalGlue())

        val saveButton = JButton("Save")

        add(saveButton)
    }
}